api/preprints/serializers.py

```diff
@@ -76,6 +77,10 @@ class PreprintSerializer(JSONAPISerializer):
     is_preprint_orphan = ser.BooleanField(read_only=True)
     license_record = NodeLicenseSerializer(required=False, source='license')
 
+    reviews_state = ser.CharField(read_only=True, max_length=15)
+
+    submit_for_review = ser.BooleanField(required=False, write_only=True)
```

Comment:
On the preprint create/edit pages, every time you "save and continue", this endpoint is hit, but we don't want to [re]submit until they click the big green button at the end. I added this because right now there's no way to tell the difference between the big green button and "save and continue" with no changes.